### PR TITLE
Master

### PR DESCRIPTION
**Pull request by jjoksch**

### CODE REVIEW

**Comment by cpaulik** [comment]
Matrikelnummer bitte.

**Comment by jjoksch** [comment]
Hallo!
Meine Matrikelnummer lautet: 0725536
lg

File: ex1/dates_times.py [hunk 1]
```diff
@@ -30,15 +30,57 @@
...
 # If ref is before dt then use 'since' instead of 'until'
+
+def how_long(dt, ref):
+    '''
+    :param dt: datetime
+    :param ref: refference datetime
+    :return: the differnece between both imputs in days, minutes and seconds
+    '''
+
+    d = ref - dt
```
Comment (cpaulik) [inline ex1/dates_times.py:73]: Here you should have changed the direction of the comparison depending on which datetime is sooner.

File: ex1/functions.py [hunk 2]
```diff
@@ -91,25 +192,116 @@ def f():
...
+    :param l: list
+    :return: a dictionary with all arguments of l with each position
+    '''
+
+    result = {}
+    i = 0
+
+    for j in l:
+        result[i] = j
+        i += 1
```
Comment (cpaulik) [inline ex1/functions.py:221]: always try to use `enumerate`

**Pull request closed without merging**